sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/ShuffleExchange.scala

```diff
@@ -47,7 +47,7 @@ case class ShuffleExchange(
     val extraInfo = coordinator match {
       case Some(exchangeCoordinator) =>
         s"(coordinator id: ${System.identityHashCode(exchangeCoordinator)})"
-      case None => ""
+      case _ => ""
```

Comment:
should we adjust this pattern match condition at other places in the class below as well?